1-estrategia/personasux.md

```diff
@@ -0,0 +1,66 @@
+
+Pero el personaje no tiene que ser exactamente correcto, tiene que ser preciso. Es decir, no es necesario que hacer fundarse en promedios para asegurarse de la exactitud de los personajes. Sus características son hipotéticas pero específicas. Estos personajes permiten ver el alcance y la naturaleza del problema de diseño. Son únicos para cada proyecto. 
+
+Después haber determinado los personajes del proyecto es necesario que las otras personas que trabajan en la interfaz se familiarizan con ellos y la denominación "personaje". Sobre todo los programadores que desarrollan el producto pero que no están habituados a referirse a los personajes. Prefieran hablar de los "usuarios" cuando hacemos referencia a las personas a las que se dedica la interfaz. Sin embargo, este término es demasiado general para entender cómo diseñar una buena interfaz. Cuando utilizamos "usuarios" no es preciso y no es especificado porque nos referimos a un conjunto de personas no determinadas específicamente, sin nombres, sin características y sin caras. Hablamos de "los que van a utilizar nuestra interfaz" pero no de "Marta" o "Alberto". 
+
+Por eso, habría que familiarizar los programadores a la denominación "personajes", a lo que sirven y a su valor añadido. 
+
+Notamos que los personajes son poderosos porque eliminan los debates sobre las características. En efecto, son como una herramienta de comunicación entre las personas que trabajan en el proyecto y un reflector de las decisiones tomadas. Facilitan la comprensión del diseño elegido y representan argumentos poderosos para defender el proyecto ante los programadores y los gerentes. 
+
+## Ejemplo: el P@ssport de Sony Trans Com - Alan Cooper en *Presos de la tecnología*
+
+Sony es la encargada del diseño y manufacturación de los sistemas de entretenimiento de los vuelos, el problema que tenía Sony era de diseño ya que habían creado una interfaz que consistía en un árbol jerárquico de pantallas, por el que el usuario tenía que navegar llamado “consentimiento sin información”. Esta interfaz contenía muy poca información y muchos pasos a seguir para poder visualizar el contenido elegido. Debido a sus limitaciones de prototipo había que buscar una solución.
+Lo primero que se tuvo que realizar fue una profunda investigación y para ello entrevistaron a la plantilla de Sony, de esta forma lograrían saber con gran precisión que quería lograr Sony TransCom con el producto. También realizaron diferentes entrevistas al personal de diferentes aerolíneas. Crearon personajes sin cesar, cada vez que un asistente de vuelo contaba una historia nueva creaban un nuevo personaje y en total fueron 30 los personajes creados. Al comenzar con la investigación se produjeron  similitudes entre los personajes. Aquellos personajes que tenían metas comunes se reducían a uno solo y por lo tanto fueron reduciendo hasta llegar a 10 personajes, cuatro de ellos eran pasajeros y seis tripulantes. 
```

Comment:
Aquí (y en los demás párrafos de este apartado) tenéis que añadir una línea más (son dos saltos de línea) para indicar un párrafo.